genie/genie_elt.py

```diff
@@ -0,0 +1,80 @@
+"""GENIE ELT pipeline"""
+import os
+
+from dotenv import dotenv_values
+import pandas as pd
+import snowflake.connector
+from snowflake.connector.pandas_tools import write_pandas
+import synapseclient
+
+
+def main():
+    """GENIE ELT pipeline"""
+    syn = synapseclient.login()
+
+    config = dotenv_values("../.env")
+
+    ctx = snowflake.connector.connect(
```

Comment:
Do you need to handle any context management here to close the connection? Not sure how this piece works internally.